test/libyul/objectCompiler/subObjectAccess.yul

```diff
@@ -0,0 +1,104 @@
+object "A" {
+  code {
+    pop(dataoffset("A"))
+    pop(datasize("A"))
+    pop(dataoffset("B"))
+    pop(datasize("B"))
+    pop(dataoffset("B.C"))
+    pop(datasize("B.C"))
+    pop(dataoffset("B.E"))
+    pop(datasize("B.E"))
+    pop(dataoffset("B.C.D"))
+    pop(datasize("B.C.D"))
+  }
+
+  data "data1" "Hello, World!"
+
+  object "B" {
+    code {
+      pop(dataoffset("C"))
+      pop(datasize("C"))
+      pop(dataoffset("E"))
+      pop(datasize("E"))
+      pop(dataoffset("C.D"))
+      pop(datasize("C.D"))
+    }
+    object "C" {
+      code {
+        pop(dataoffset("D"))
+        pop(datasize("D"))
+      }
+      object "D" {
+        code {
+          invalid()
+        }
+      }
+    }
+    object "E" {
+      code {
+        invalid()
+      }
+    }
+  }
+}
+// ----
+// Assembly:
+//     /* "source":26:46   */
+//   pop(0x00)
+//     /* "source":51:69   */
+//   pop(bytecodeSize)
+//     /* "source":74:94   */
+//   pop(dataOffset(sub_0))
+//     /* "source":99:117   */
+//   pop(dataSize(sub_0))
+//     /* "source":122:144   */
+//   pop(dataOffset(sub_18446744073709551615))
+//     /* "source":149:169   */
+//   pop(dataSize(sub_18446744073709551614))
+//     /* "source":174:196   */
+//   pop(dataOffset(sub_18446744073709551613))
+//     /* "source":201:221   */
+//   pop(dataSize(sub_18446744073709551612))
+//     /* "source":226:250   */
+//   pop(dataOffset(sub_18446744073709551611))
```

Comment:
This is a bit ugly. In order to get `sub_x.sub_y` we would need to access path somehow. If this is not too big problem, I would leave as is

Reply:
Fixed this to `sub_x.sub_y`